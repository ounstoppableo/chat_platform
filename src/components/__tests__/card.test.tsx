import {create} from 'react-test-renderer';
import Card from '../card/card';

it('renders correctly', () => {
  const tree = create(<Card />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});