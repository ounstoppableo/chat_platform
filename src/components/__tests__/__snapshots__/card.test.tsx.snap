// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="tw-bg-lightGray"
  style={
    {
      "height": "80%",
      "width": "80%",
    }
  }
/>
`;
